Update/some_003.txt

```diff
@@ -2887,7 +2887,7 @@ void main()
 //r私はとっさに、『面会時間外だから使わせてもらった』と言い訳を考え、後ろめたい思いだったけど藤堂くんにそう説明しようと口を開きかけた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "私はとっさに、『面会時間外だから使わせてもらった』と言い訳を考え、後ろめたい思いだったけど藤堂くんにそう説明しようと口を開きかけた。",
-			NULL, "To my disappointment, I thought of the excuse \"I used it because it's outside of visiting hours.\" I thought it was awkward, but I opened my mouth to explain it to Toudou-kun.", Line_Normal);
+			NULL, "I thought of the excuse \"I used it because it's past of visiting hours.\" I felt guilty, but opened my mouth to explain it to Toudou-kun.", Line_Normal);
```

Comment:
```suggestion
			NULL, "I thought of the excuse \"I used it because it's past visiting hours.\" I felt guilty, but opened my mouth to explain it to Toudou-kun.", Line_Normal);
```